cmake/horace_CPackConfig.cmake

```diff
@@ -0,0 +1,16 @@
+# For the list of CPack variables see https://cmake.org/cmake/help/v3.7/module/CPack.html
+
+set(CPACK_RESOURCE_FILE_LICENSE "${CMAKE_SOURCE_DIR}/LICENSE")
+set(CPACK_RESOURCE_FILE_README "${CMAKE_SOURCE_DIR}/README.md")
+set(CPACK_PACKAGE_VERSION_MAJOR "${${PROJECT_NAME}_VERSION_MAJOR}")
+set(CPACK_PACKAGE_VERSION_MINOR "${${PROJECT_NAME}_VERSION_MINOR}")
+set(CPACK_PACKAGE_VERSION_PATCH "${${PROJECT_NAME}_VERSION_PATCH}")
+set(CPACK_PACKAGE_FILE_NAME "${PROJECT_NAME}-${${PROJECT_NAME}_FULL_VERSION}")
```

Comment:
the Herbert equivalent uses 
```
set(CPACK_PACKAGE_FILE_NAME "${PROJECT_NAME}-${Herbert_FULL_VERSION}")
```
and you refer to Horace_FULL_VERSION in the `horace_Version.cmake` script

Reply:
Which do you think is best using `${horace_FULL_VERSION}` or `${${PROJECT_NAME}_FULL_VERSION}`.

The second is definitely more general - so when we come to extracting out common CMake functions, we can use the latter. But, at this point there's no practical distinction. Unless you think it's better to be consistent between the two repos? Or we could raise an issue to alter Herbert

Reply:
it's more consistent to be using `$PROJECT_NAME` in this file; it is much less readable though, so definitely keep `Horace_` in the other scripts.

I'd suggest small PR in Herbert to make the scripts consistent once Horace is finished.